docs/docs/developers/docs/aztec-nr/framework-description/functions/attributes.md

```diff
@@ -21,53 +21,42 @@ A private function operates on private information, and is executed by the user
 
 `#[external("private")]` is just syntactic sugar. At compile time, the Aztec.nr framework inserts code that allows the function to interact with the [kernel](../../../foundational-topics/advanced/circuits/kernels/private_kernel.md).
 
-To help illustrate how this interacts with the internals of Aztec and its kernel circuits, we can take an example private function, and explore what it looks like after Aztec.nr's macro expansion.
+If you are interested in what exactly the macros are doing we encourage you to run `nargo expand` on your contract.
+This will display your contract's code after the transformations are performed.
```

Comment:
This can be seen as regression as now devs need to run `nargo expand` manually but I think it's the better tradeoff because they will get to see a fresh code and not the stale thing that was in `DocsExample` contract.

Also these docs are now very old and I am not sure if they'll even survive for long.